receiver/vcenterreceiver/documentation.md

```diff
@@ -325,6 +325,22 @@ The amount of memory that is ballooned due to virtualization.
 | ---- | ----------- | ---------- | ----------------------- | --------- |
 | By | Sum | Int | Cumulative | false |
 
+### vcenter.vm.memory.ssdswapped
+
+The amount of memory swapped to fast disk device such as SSD.
+
+| Unit | Metric Type | Value Type | Aggregation Temporality | Monotonic |
+| ---- | ----------- | ---------- | ----------------------- | --------- |
+| KiBy | Sum | Int | Cumulative | false |
```

Comment:
This metric unit is KiBy, different from other metric which is MiBy. Is this designed for some other purposes?

Reply:
That is the unit or the source data https://vdc-repo.vmware.com/vmwb-repository/dcr-public/790263bc-bd30-48f1-af12-ed36055d718b/e5f17bfc-ecba-40bf-a04f-376bbb11e811/vim.vm.Summary.QuickStats.html

> The amount of memory swapped to fast disk device such as SSD, in KB.

We could massage the data, however usually I prefer to keep the same unit of the source of the data